### PR TITLE
feat: bulk order module

### PR DESCRIPTION
**Pull request by joao-conde**
Adds the bulk order module endpoints to the RIPE API for feature parity with the JS SDK.

Also adds the pylint config that already catched a bug in a method (hail linters)

### CODE REVIEW

**Review by BeeMargarida — APPROVED** [review]

**Review by 3rdvision — APPROVED** [review]

File: src/ripe/bulk_order.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        contents = self.get(url, **kwargs)
+        return contents
+
+    def create_attachment_bulk_order(self, number, files, **kwargs):
+        url = self.base_url + "bulk_orders/%d/attachments" % number
+        data_m = dict(files = files)
+        contents = self.post(url, data_m = data_m, **kwargs)
+        return contents
+
+    def _import_bulk_order(self, name, orders, **kwargs):
```
Comment (joamag) [inline src/ripe/bulk_order.py:65]: why the separate method?

**Review by joamag — CHANGES_REQUESTED** [review]

File: src/ripe/bulk_order.py (hunk 1 above)
Comment (joao-conde) [reply src/ripe/bulk_order.py:65]: Following JS SDK's logic: 
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/bulk-order.js#L428
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/bulk-order.js#L532

And to a certain extent the order module too:
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/order.js#L1711
- https://github.com/ripe-tech/ripe-sdk/blob/master/src/js/api/order.js#L2001

Comment (joamag) [reply src/ripe/bulk_order.py:65]: let's not copy these practices from RIPE SDK as it has a very specific way of handling code structure

**Review by joamag — CHANGES_REQUESTED** [review]

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@joao-conde removed some kwargs from methods as they were not needed, let's try to keep the method signatures as simple as possible

**Pull request merged**